square_side: 3.0
circle_radius: 2.0
triangle_side: 3.0
